### DISCUSSION TITLE
Test Replay Support

### DISCUSSION THREAD

**Discussion by ivan-xue-prospa** [currents-dev/cypress-currents-reporter#181] [General]
Cypress v13 supports Test Replay in Cypress Cloud, will currents support this in the future? 

Another important feature of Cypress App is local Debug for CI tests, is there any plan to support this as well?

**Comment by agoldis** [comment]
> Another important feature of Cypress App is local Debug for CI tests, is there any plan to support this as well?

What do you mean?

> Cypress v13 supports Test Replay in Cypress Cloud, will currents support this in the future?

We released our own cypress-debugger (https://cypress-debugger.dev) back in May 2023 - which is a standalone, free, open-source plugin. Test Replay provides the same functionality as cypress-debugger. 

Note, that Test Replay is a closed-source software - it isn't a part of Cypress App and we cannot build on it unless Cypress.io changes the license. Read more here: https://currents.dev/posts/currents-cypress-replay-debugger
